Definitions/Chord

```diff
@@ -0,0 +1,29 @@
+class Chord:
+    def __init__(self, symbol, degrees, next, inversion):
+        self.symbol = symbol
+        self.degrees = degrees
+        self.next = next
+        self.inversion = inversion
+        
+    def hasMelodicNote(self, note):
```

Comment:
sytactically, might change this to `hasScaleDegree`.

Also, no need to save result to variable if you're just then going to return it.